cmd/cli/kubectl-kyverno/utils/common/common_test.go

```diff
@@ -95,12 +96,12 @@ func Test_NamespaceSelector(t *testing.T) {
 			},
 		},
 	}
-
+	var dClient dclient.Interface
 	rc := &ResultCounts{}
 	for _, tc := range testcases {
 		policyArray, _ := yamlutils.GetPolicy(tc.policy)
 		resourceArray, _ := GetResource(tc.resource)
-		ApplyPolicyOnResource(policyArray[0], resourceArray[0], "", false, nil, v1beta1.RequestInfo{}, false, tc.namespaceSelectorMap, false, rc, false, nil)
+		ApplyPolicyOnResource(policyArray[0], resourceArray[0], "", false, nil, v1beta1.RequestInfo{}, false, tc.namespaceSelectorMap, false, rc, false, nil, dClient)
```

Comment:
add another struct variable for all parameters here?

Reply:
fixed.